packages/preview/ssrn-scribe/0.6.0/README.md

```diff
@@ -0,0 +1,114 @@
+
+## How to use
+
+### Use as a template package
+
+Typst integrated the template with their official package manager. You can use it as the other third-party packages.
+
+You only need to enter the following command in the terminal to initialize the template.
+
+```bash
+typst init @preview/ssrn-scribe
+```
+
+If will generate a subfolder `ssrn-scribe` including the `main.typ` file in the current directory with the latest version of the template.
+
+### Mannully use
+
+1. Download the template or clone the repository.
+2. generate your bibliography file using `.biblatex` and store the file in the same directory of the template.
+3. modify the `main.typ` file in the subfolder `/template` and compile it.
+   ***Note:* You should have `paper_template.typ` and `main.typ` in the same directory.**
+
+In the template, you can modify the following parameters:
+
+`maketitle` is a boolean (**compulsory**). If `maketitle=true`, the template will generate a new page for the title. Otherwise, the title will be shown on the first page.
+
+- `maketitle=true`:
+
+| Parameter | Default | Optional | Description |
+| --- | --- | --- | --- |
+| `font` | "PT Serif" | Yes | The font of the paper. You can choose "Times New Roman" or "Palatino" |
+| `fontsize` | 11pt | Yes | The font size of the paper. You can choose 10pt or 12pt |
+| `title` | "Title" | No | The title of the paper |
+| `subtitle` | none | Yes | The subtitle of the paper, use "" or [] |
+| `authors` | none | No | The authors of the paper |
+| `date` | none | Yes | The date of the paper |
+| `abstract` | none | Yes | The abstract of the paper |
+| `keywords` | none | Yes | The keywords of the paper |
+| `JEL` | none | Yes | The JEL codes of the paper |
+| `acknowledgments` | none | Yes | The acknowledgment of the paper |
+| `bibliography` | none | Yes | The bibliography of the paper ```bibliography: bibliography("bib.bib", title: "References", style: "apa")``` |
+
+- `maketitle=false`:
+
+| Parameter | Default | Optional | Description |
+| --- | --- | --- | --- |
+| `font` | "PT Serif" | Yes | The font of the paper. You can choose "Times New Roman" or "Palatino" |
+| `fontsize` | 11pt | Yes | The font size of the paper. You can choose 10pt or 12pt |
+| `title` | "Title" | No | The title of the paper |
+| `subtitle` | none | Yes | The subtitle of the paper, use "" or [] |
+| `authors` | none | No | The authors of the paper |
+| `date` | none | Yes | The date of the paper |
+| `bibliography` | none | Yes | The bibliography of the paper ```bibliography: bibliography("bib.bib", title: "References", style: "apa")``` |
+
+**Note: You need to keep the comma at the end of the first bracket of the author's list, even if you have only one author.**
+
+```typst
+    (
+    name: "",
+    affiliation: "", // optional
+    email: "", // optional
+    note: "", // optional
+    ),
+```
+
+```typst
+#import "@preview/ssrn-scribe:0.5.0": *
```

Comment:
You probably want to update this import.

Reply:
I have updated the version in readme file.